src/flowMC/resource/model/common.py

```
@@ -213,25 +217,24 @@ def inverse(
     shift_MLP: MLP
-    dt: Float = 1
+    dt: float = 1
 
-    def __init__(self, scale_MLP: MLP, shift_MLP: MLP, dt: Float = 1):
+    def __init__(self, scale_MLP: MLP, shift_MLP: MLP, dt: float = 1):
         self.scale_MLP = scale_MLP
         self.shift_MLP = shift_MLP
         self.dt = dt
 
     def __call__(
         self, x: Float[Array, " n_dim"], condition_x: Float[Array, " n_cond"]
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Expected an expression

(F722)